src/App.jsx

```diff
@@ -3,15 +3,30 @@ import Header from './commonComponents/Header/Header';
 import AppWrapper from './commonComponents/AppWrapper/AppWrapper';
 import Search from './commonComponents/Search/Search';
 import './sass/App.sass';
+import {BrowserRouter, Route, Switch} from "react-router-dom";
+import BeerPage from "./commonComponents/BeerPage/BeerPage";
+import FavoritesPage from "./commonComponents/FavoritesPage/FavoritesPage";
```

Comment:
it's better to separate pages from components. Also, only common components for all pages should be in the commonComonents folder.  Components that are related only to one page should be in the folder components for the target page

Reply:
Done